src/styles/index.scss

```diff
@@ -1,3 +1,110 @@
 body {
+  color: rgba(6, 11, 53, 1);
   margin: 0;
+  font-family: Roboto, sans-serif;
+  line-height: 14px;
+}
+
+.product-card {
+  width: 200px;
+  box-sizing: border-box;
+  background: white;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+}
```

Comment:
Consider using SCSS nesting for elements, modifiers, and pseudo-classes within the `.product-card` block. This will improve the maintainability and readability of your styles.